packages/pn-personafisica-webapp/cypress/e2e/Delegate.cy.ts

```diff
@@ -13,51 +18,135 @@ describe('Delegation', () => {
+        
+        // cy.get('[data-testid="loading-spinner"] > .MuiBox-root').should('not.exist');
+        cy.wait('@notificationDetailAsDelegate').then((interception) => {
+          expect(interception.request.url).include(`mandateId=${mandateId}`);
+          expect(interception.request.url).include(iun);
+          expect(interception.response.statusCode).to.equal(200);
+        });
       });
     });
+
+    it('Should reject delegator\'s request', () => {
+      cy.get('[data-testid="menu-list"] > :nth-child(4) > .MuiBox-root').contains(/1/).should('exist');
+      cy.visit(DELEGHE);
+      cy.wait('@getDelegators');
+      cy.get(':nth-child(3) > .css-1gi4qli > .MuiTableContainer-root > [data-cy="table(notifications)"] > .MuiTableBody-root > :nth-child(1) > .css-1ogoim7-MuiTableCell-root > [data-testid="delegationMenuIcon"] > [data-testid="MoreVertIcon"]').click();
+      cy.get('.MuiPaper-root > .MuiList-root > .MuiButtonBase-root').click();
+      cy.intercept('PATCH', `${REJECT_DELEGATION('af02d543-c67e-4c64-8259-4f7ac12249fd')}`, {
+        statusCode: 200
+      }).as('rejectDelegation');
+      cy.get('.css-pj2eij-MuiGrid-root > [data-testid="dialogAction"]').click();
+      cy.wait('@rejectDelegation');
+      cy.get('.css-19kzrtu > :nth-child(3)').contains(/Giuseppe Maria Garibaldi/).should('not.exist');
+    });
+
+    it('Should accept delegator\s request', () => {
+      cy.get('[data-testid="menu-list"] > :nth-child(4) > .MuiBox-root').contains(/1/).should('exist');
+      cy.visit(DELEGHE);
+      cy.wait('@getDelegators');
+      cy.get('.MuiTableCell-alignCenter > .MuiButtonBase-root').click();
+      const verificationCode = '25622';
+      verificationCode.split('').forEach((code, i) =>
+        cy.get(`:nth-child(${i + 1}) > .MuiInputBase-root > #outlined-basic`).type(code)
+      );
+      cy.intercept(`${DELEGATIONS_BY_DELEGATE()}`, {
+        fixture: 'delegations/mandates-by-delegate-after-activation',
+      }).as('getDelegatesAfterActivation');
+      cy.intercept('PATCH',`${ACCEPT_DELEGATION('af02d543-c67e-4c64-8259-4f7ac12249fd')}`, {
+        statusCode: 204
+      }).as('acceptDelegation');
+      cy.get('.MuiDialogActions-root > .MuiButton-contained').click();
+      cy.wait('@getDelegatesAfterActivation');
+      cy.get(':nth-child(2) > .MuiTableCell-alignCenter > .MuiChip-root > .MuiChip-label').contains(/Attiva/).should('exist');
+    });
+  });
+
+  describe('As delegator', () => {
+    it('Should create new delegate', () => {
+      cy.visit(DELEGHE);
+      cy.get('.css-1teipz8-MuiStack-root > .MuiBox-root > .MuiButtonBase-root').click();
+      const formDelegator = {
+        name: 'Cristoforo',
+        surname: 'Colombo',
+        fiscalCode: 'CLMCST42R12D969Z',
+      }
+      cy.get('#nome').type(formDelegator.name);
+      cy.get('#cognome').type(formDelegator.surname);
+      cy.get('#codiceFiscale').type(formDelegator.fiscalCode);
+      cy.intercept('POST', `${CREATE_DELEGATION()}`, {
```

Comment:
You should not only intercept this request but also verify it is sent properly. Check the payload is what we expect!